doc/rbd/rbd-nomad.rst

```diff
@@ -52,20 +52,24 @@ diagram depicts the Nomad/Ceph technology stack.
 Create a Pool
 =============
 
-By default, Ceph block devices use the ``rbd`` pool. Create a pool for
-Nopmad persistent storage. Ensure your Ceph cluster is running, then create
-the pool. ::
+By default, Ceph block devices use the ``rbd`` pool. Create a pool for Nomad
+persistent storage. Ensure that your Ceph cluster is running, then create the
+pool:
 
-        $ ceph osd pool create nomad
+.. prompt:: bash #
```

Comment:
i think almost all ceph commands do not require root privilege unless we need to access / write to files owned by root.

Reply:
True, I just usually use root user and kept it in documentation.

Reply:
Yours was actually right.  Zac's `prompt` change makes it look like root is required which is not the case.

Reply:
I'm changing it back now. My bad.